pydantic/main.py

```diff
@@ -673,6 +681,11 @@ def _get_value(
                 and (not value_include or value_include.is_included(i))
             )
 
+        elif getattr(cls.Config, 'use_enum_values', False) and any(
+            is_literal_type(v) for v in get_type_hints(cls).values()
```

Comment:
This will break if you have a second field in your model e.g.
```python
    class Model(BaseModel):
        baz: Literal[FooEnum.foo]
        boo: str
```
It will run `v.value` on `boo`
(And btw you should use another variable instead of a local `v`, which is confusing with the other one)